### ISSUE TITLE
Accents and special characters in rows of a Table breaks "CSV Download"

### ISSUE THREAD

**Issue by firasm** [abelardopardo/ontask_b#98]
This issue may be related to issue #43 ...

Steps to reproduce:

- Copy the following three rows into a csv file, then import into a new workflow:

```
Student Number,First,Last Namé
1111,Bélabcde,Ctrahil
2222,Eyhrg,Boë
3333,Larzan,Tgdrée
```

- Once the import is complete, click "CSV Download" to try and download the table

A system error is thrown, but the user facing logs do not show anything and no log is created to debug the issue.

Suggested fix:

- Throw an error upon data import (bad)
- Remove/strip/replace all non-supported characters or (better but still bad)
- Handle these characters properly (ideal) 

No matter which fix is implemented, errors should be handled a little better with some more logging and notifications to the end-user about where to troubleshoot

**Comment by abelardopardo** [comment]
Confirmed. This is related to issue #43, and I think the fix is now more robust. It'll be pushed in the next release.

**Comment by abelardopardo** [comment]
Release B.3.0.3 now fixes these three issues.

**Comment by firasm** [comment]
Hey @abelardopardo, thanks for all the fixes!

We've updated to 3.0.3 but when I try to upload the same CSV file that I submitted as part of my bug report for issue #99, I get an error indicating that utf8-8 is an unknown encoding.

What encoding should we now be using?

<img width="575" alt="utf8" src="https://user-images.githubusercontent.com/2507459/47052639-14268580-d15e-11e8-950f-2a0a31526d89.png">
